core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -184,6 +185,13 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
           error("whitelist must be specified when using new consumer in mirror maker.")
           System.exit(1)
         }
+
+        if (!Utils.loadProps(options.valueOf(consumerConfigOpt)).keySet().contains(NewConsumerConfig.PARTITION_ASSIGNMENT_STRATEGY_CONFIG))
```

Comment:
nit: This line seems a little wasteful. Seems like we could store the loaded properties in a variable and pass it down to `createOldConsumers` and `createNewConsumers` directly instead of the path.